packages/neos-ui-guest-frame/src/InlineUI/index.js

```diff
@@ -48,6 +48,8 @@ export default class InlineUI extends PureComponent {
         const canBeDeleted = $get('policy.canRemove', this.props.focusedNode) || false;
         const canBeEdited = $get('policy.canEdit', this.props.focusedNode) || false;
         const visibilityCanBeToggled = !$contains('_hidden', 'policy.disallowedProperties', this.props.focusedNode);
+        // Check if focusedNode is in Guest frame to prevent errors (e.g. if still in store)
+        const focusedNodeInGuestFrame = findNodeInGuestFrame(focusedNodeContextPath) ? focused : {};
```

Comment:
Just one last question. Shouldn't we just `return null` in case there's no `focusedNodeInGuestFrame`, instead of providing it with a dummy empty object instead of the node?

Reply:
Good point, if we dont have the toolbar, we dont need the container